### PR TITLE
[ci] update active branches - (49)

### PR DESCRIPTION
**Pull request by ankitkacn**
### Motivation
Update active branches.

### Test Plan
CI

### CODE REVIEW

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -147,7 +147,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target-branch: [latest, release-1.2, release-1.3]
+        target-branch: [main, release-1.3, release-1.4]
```
Comment (dhaneshacn) [inline .github/workflows/daily.yml]: keep the name of the branch as latest, don't change it

File: .github/workflows/daily.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         #this is a painful solution since it doesn't pick up new branches, other option is lotsa shell in one job....
         #to test in canary add in canary here.....
-        target-branch: [latest, release-1.2, release-1.3]
+        target-branch: [main, release-1.3, release-1.4]
```
Comment (dhaneshacn) [inline .github/workflows/daily.yml]: keep the name of the branch as latest, don't change it

**Comment by ankitkacn** [comment]
Closing this PR as this PR have a technical problem, I am going to create new PR with the same name and that is working fine. 
https://github.com/diem/diem/pull/10558

**Pull request closed without merging**